src/about/About.tsx

```diff
@@ -0,0 +1,67 @@
+import './About.css';
+import { Link } from 'react-router-dom';
+
+function About() {
```

Comment:
А почему предпочли function declaration, а не стрелочные?
Со стрелочными все выглядит гораздо проще
```suggestion
export const About = () => (
  <div className="about-page">
    ...
  );

```
И я по-прежнему очень рекомендую использовать именованные импорты)